genref/config.yaml

```diff
@@ -0,0 +1,145 @@
+hiddenMemberFields:
+  - "TypeMeta"
+
+externalPackages:
+  - match: ^k8s\.io/apimachinery/pkg/apis/meta/v1\.Duration$
+    target: https://godoc.org/k8s.io/apimachinery/pkg/apis/meta/v1#Duration
+  - match: ^k8s\.io/apimachinery/pkg/types\.UID$
+    target: https://godoc.org/k8s.io/apimachinery/pkg/types#UID
+  - match: ^k8s\.io/apimachinery/pkg/runtime\.RawExtension$
+    target: https://godoc.org/k8s.io/apimachinery/pkg/runtime/#RawExtension
+
+  - match: ^k8s\.io/apimachinery/pkg/runtime\.Unknown$
+    target: https://godoc.org/k8s.io/apimachinery/pkg/runtime#Unknown
+  - match: ^time\.Duration$
+    target: https://godoc.org/time#Duration
+  - match: ^k8s\.io/(api|apimachinery/pkg/apis)/
+    target: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#{{- lower .TypeIdentifier -}}-{{- arrIndex .PackageSegments -1 -}}-{{- arrIndex .PackageSegments -2 -}}
```

Comment:
Why is the v1.19 reference here?

Reply:
Good spot. This should be v1.20. The whole thing was implemented before v1.20 was out.

Reply:
I am not clear why the `kubernetes.io/docs/reference/...` target is included?

Reply:
Take `k8s.io/apimachinery/pkg/apis` as an example, there are many fundamental data types or structs defined. Parsing that whole package is tedious and costly. Referencing the existing docs may save us some efforts.